aiida/cmdline/params/options/commands/code.py

```diff
@@ -15,6 +15,14 @@
 from aiida.cmdline.params.options.overridable import OverridableOption
 
 
+def is_on_container(ctx):
```

Comment:
I would say either `is_in_container` or `is_containerized` (the latter would be what one says in practice)